hotsos/defs/tests/scenarios/openstack/ovsdb_server_frequent_reelections.yaml

```diff
@@ -0,0 +1,14 @@
+target-name: ovsdb_server_frequent_reelections.yaml
+data-root:
+  files:
+    var/log/neutron/neutron-ovn-metadata-agent.log: |
+      2022-08-18 12:30:36.103 75556 ERROR ovsdbapp.backend.ovs_idl.transaction [-] OVSDB Error: no error details available
```

Comment:
nit: the tests data_root has:

```
$ cat tests/unit/fake_data_root/openstack/sos_commands/date/date
Thu Feb 10 16:19:17 UTC 2022
```

so while the timestamp you have added will work, it is in fact several months in the future of the date. this is just a nit but to bring it inline you could do:

```
2022-02-10 12:30:36.103 75556 ERROR ovsdbapp.backend.ovs_idl.transaction [-] OVSDB Error: no error details available
```

Reply:
has changed to 2022-02-10